### PR TITLE
Fix RTD

### PR DESCRIPTION
**Pull request by astrojuanlu**
### Description of Change
*Replace this text with why this change is required and how it was accomplished*

### Assumptions
*Replace this text with any assumptions made (if any)*

### Checklist for All Submissions
* [ ] I have added change info to [CHANGES.rst](https://github.com/aio-libs/aiobotocore/blob/master/CHANGES.rst)
* [ ] If this is resolving an issue (needed so future developers can determine if change is still necessary and under what conditions) (can be provided via link to issue with these details):
  * [ ] Detailed description of issue
  * [ ] Alternative methods considered (if any)
  * [ ] How issue is being resolved
  * [ ] How issue can be reproduced
* [ ] If this is providing a new feature  (can be provided via link to issue with these details):
  * [ ] Detailed description of new feature
  * [ ] Why needed
  * [ ] Alternatives methods considered (if any)

### Checklist when updating botocore and/or aiohttp versions

* [ ] I have read and followed [CONTRIBUTING.rst](https://github.com/aio-libs/aiobotocore/blob/master/CONTRIBUTING.rst#how-to-upgrade-botocore)
* [ ] I have updated test_patches.py where/if appropriate (also check if no changes necessary)
* [ ] I have ensured that the awscli/boto3 versions match the updated botocore version

### CODE REVIEW

**Review by thehesiod — APPROVED** [review]
i have no idea how this works, but if it passes I'll merge ;)

**Comment by astrojuanlu** [comment]
Almost there :)

**Comment by astrojuanlu** [comment]
Done! https://aiobotocore--1094.org.readthedocs.build/en/1094/

File: docs/conf.py [hunk 1]
```diff
@@ -126,7 +126,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path = ['_static']
```
Comment (thehesiod) [inline docs/conf.py:129]: commented out code?

**Comment by thehesiod** [comment]
thank you! just one comment

File: docs/conf.py (hunk 1 above)
Comment (astrojuanlu) [reply docs/conf.py:129]: I can remove it. Should I remove the preceding paragraph too?

Comment (astrojuanlu) [reply docs/conf.py:129]: (The "Add any paths" explanation)

Comment (astrojuanlu) [reply docs/conf.py]: Went ahead and did it

Comment (thehesiod) [reply docs/conf.py]: yes pls

**Pull request merged**